datastore/query.go

```diff
@@ -651,6 +751,15 @@ func (c *Client) GetAll(ctx context.Context, q *Query, dst interface{}) (keys []
 	ctx = trace.StartSpan(ctx, "cloud.google.com/go/datastore.Query.GetAll")
 	defer func() { trace.EndSpan(ctx, err) }()
 
+	res, err := c.GetAllWithOptions(ctx, q, dst)
+	return res.Keys, err
+}
+
+// GetAllWithOptions is similar to GetAll but runs the query with provided options
+func (c *Client) GetAllWithOptions(ctx context.Context, q *Query, dst interface{}, opts ...RunOption) (res GetAllWithOptionsResult, err error) {
```

Comment:
is it possible for opts to be empty here?

Reply:
Yes. A possible call to GetAllWithOptions is:
```
var got []SQChild
gotRes, gotErr := client.GetAllWithOptions(ctx, query, &got)
```